tenable_io/client.py

```diff
@@ -115,9 +121,13 @@ def wrapper(*args, **kwargs):
 
             if response.status_code == 429:
                 raise TenableIORetryableApiException(response)
-            if response.status_code in [501, 502, 503]:
+            if response.status_code in [501, 502, 503, 504]:
                 raise TenableIORetryableApiException(response)
             if not 200 <= response.status_code <= 299:
+                logging.error(u'Response %s %s %s %s %s %s' % (response.request.method, response.url, response.reason,
```

Comment:
We should keep all the error logging in the `_request` method, use `logging.error` when `response.status_code` is not `2XX` in additional to the `logging.debug`.